subreddit.py

```diff
@@ -15,12 +15,17 @@
                      user_agent=config.user_agent_new)
 
 subreddits_old_account = list(old_account.user.subreddits(limit=None))
+
+
 print(f'Old account subscribed to {len(subreddits_old_account)} subreddits.\n\n')
 
+
 subreddits_new_account = list(new_account.user.subreddits(limit=None))
+print(f'New account subscribed to {len(subreddits_new_account)} subreddits.\n\n')
```

Comment:
Could you please change this to 

`print(f'New account is subscribed to {len(subreddits_new_account)} subreddits before starting the migration.\n\n')`